.travis.yml

```diff
@@ -8,43 +8,34 @@ language: perl
 perl:
   - blead
   - dev
-  - '5.28'
-  - '5.26'
-  - '5.24'
-  - '5.22'
-  - '5.20'
-  - '5.18'
-  - '5.16'
-  - '5.14'
-  - '5.12'
-  - '5.10'
+  - "5.30"
+  - "5.28"
+  - "5.26"
+  - "5.24"
+  - "5.22"
+  - "5.20"
+  - "5.18"
+  - "5.16"
+  - "5.14"
+  - "5.12"
+  - "5.10"
 cache:
   directories:
-    - '~/perl5'
+    - "~/perl5"
 matrix:
   allow_failures:
     - perl: blead
   fast_finish: 1
   include:
     - env: COVERAGE=1
-      perl: '5.28'
+      perl: "5.30"
 env:
   global:
     - AUTHOR_TESTING=1
     - AUTOMATED_TESTING=1
     - RELEASE_TESTING=1
 before_install:
-  - git clone git://github.com/travis-perl/helpers ~/travis-perl-helpers
-  - source ~/travis-perl-helpers/init
-  - build-perl
-  - perl -V
-  - build-dist
-  - cd $BUILD_DIR
-install:
-  - cpan-install --deps
```

Comment:
Are you sure we can do without this line? The travis helpers do this?

Reply:
line 44, that is

Reply:
Yes, see the sample simple configuration: https://github.com/travis-perl/helpers#example-simple-travisyml